composer/yamls/models/glue/mnli-m.yaml

```diff
@@ -0,0 +1,74 @@
+train_dataset:
```

Comment:
What's the purpose of this `mnli-m` YAML config? Seems like a duplicate.

Reply:
Ah, it's two different datasets -- mnli matched versus mnli-mismatched. We can merge them when the "evaluate on multiple datasets" PR is merged in!